bom-latest/pom.xml

```diff
@@ -282,7 +282,7 @@
             <dependency>
                 <groupId>org.jenkins-ci.plugins</groupId>
                 <artifactId>junit</artifactId>
-                <version>1.50</version>
+                <version>1.51</version>
```

Comment:
How is this related?

Reply:
Seems to happen with dependabot sometimes, if you tell it to rebase it goes away